project/plugins.sbt

```diff
@@ -6,7 +6,7 @@ addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.5")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
 
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
```

Comment:
Why do we wanna use a release candidate?

Reply:
I just used the version referenced here: https://github.com/jrudolph/sbt-dependency-graph

Reply:
I think that's the only one available...it doesn't really matter much since this is only used locally for developer to see dependencies

Reply:
👍

Reply:
by `available` I meant that works with sbt 1.3.+